src/main/java/KarateReportPortalRunner.java

```diff
@@ -0,0 +1,139 @@
+
+        import com.intuit.karate.CallContext;
+        import com.intuit.karate.ScriptContext;
+        import com.intuit.karate.cucumber.DummyFormatter;
+        import com.intuit.karate.cucumber.DummyReporter;
+        import com.intuit.karate.cucumber.KarateFeature;
+        import com.intuit.karate.cucumber.KarateHtmlReporter;
+        import com.intuit.karate.cucumber.KarateRuntime;
+        import com.intuit.karate.cucumber.KarateRuntimeOptions;
+        import com.intuit.karate.junit4.Karate;
+        import com.intuit.karate.junit4.KarateFeatureRunner;
+        import cucumber.runtime.RuntimeOptions;
+        import cucumber.runtime.RuntimeOptionsFactory;
+        import cucumber.runtime.junit.FeatureRunner;
+        import cucumber.runtime.junit.JUnitOptions;
+        import cucumber.runtime.junit.JUnitReporter;
+        import gherkin.formatter.model.Match;
+        import gherkin.formatter.model.Result;
+        import gherkin.formatter.model.Scenario;
+        import gherkin.formatter.model.Step;
+        import java.io.IOException;
+        import java.util.ArrayList;
+        import java.util.HashMap;
+        import java.util.Iterator;import java.util.List;
+        import java.util.Map;
+        import org.junit.Test;
+        import org.junit.runner.Description;
+        import org.junit.runner.notification.RunNotifier;
+        import org.junit.runners.model.FrameworkMethod;
+        import org.junit.runners.model.InitializationError;
+        import org.slf4j.Logger;
+        import org.slf4j.LoggerFactory;
+
```

Comment:
look at the formatting of code, lines starts not from the beginning.
If you use IntelliJ Idea apply Ctrl+Alt + L to format the code.